base/genimport/importer.go

```diff
@@ -269,14 +274,93 @@ func removeAllFilesInDirExcept(o *Output, dir string, except_list []string) {
 }
 
 func createPluginGoModFile(o *Output, pkgpath string, dir string) string {
+	newLines := [][]byte{
+		[]byte("module gomacro.imports/" + pkgpath + "\n"),
+	}
+
+	// TODO: Figure out a better way to guess the dir
+	if pkgModFileInfo, err := getModuleFileInfo("."); err == nil &&
+		strings.Contains(strings.ToLower(pkgpath), strings.ToLower(pkgModFileInfo.Path)) {
```

Comment:
Go import paths are supposed to be case-sensitive, no?  Also this should be has-prefix -- really it should even be `pkgPath == modPath || HasPrefix(pkgPath, modPath + "/")`!

Reply:
Good point! Will fix!